src/clientmetricreport/GlobalMetricReport.ts

```diff
@@ -4,4 +4,5 @@
 export default class GlobalMetricReport {
   previousMetrics: { [id: string]: number } = {};
   currentMetrics: { [id: string]: number } = {};
+  stringValues: { [id: string]: string } = {};
```

Comment:
maybe `currentDimensions` or just `dimensions` instead of `stringValues`?

Reply:
I do not think they are just dimension, it can also be metric derived from string value. For example, whether decoder/encoder is in hardware is one of the derived metric.

Reply:
Updated it to `currentStringMetrics`. Also added comments that this is a workaround in case metric values are strings.